playbooks/service/swapfile.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Configure swap files
+  hosts: [ 'debops_service_swapfile', 'debops_swapfile' ]
```

Comment:
I want to drop the old groups after some time. It's just for the people to transition their inventories to the new group names.